packages/integration/src/github/SingleInstanceGithubCredentialsProvider.ts

```diff
@@ -231,19 +232,11 @@ export class GithubAppCredentialsMux {
 export class SingleInstanceGithubCredentialsProvider
   implements GithubCredentialsProvider
 {
-  static create: (
-    config: GitHubIntegrationConfig,
-  ) => GithubCredentialsProvider = config => {
-    return new SingleInstanceGithubCredentialsProvider(
-      new GithubAppCredentialsMux(config),
-      config.token,
-    );
-  };
+  static create(integrations: ScmIntegrations) {
```

Comment:
Let's avoid breaking this. We could perhaps now introduce a `DefaultGithubCredentialsProvider` with a `.fromIntegrations` factory that does this instead?

Reply:
Will do.

Reply:
Done